eslint.config.js

```diff
@@ -0,0 +1,20 @@
+const callstackConfigReact = require('@callstack/eslint-config/react.flat.js');
+
+module.exports = [
+  {
+    ignores: ['node_modules/', 'lib/'],
+  },
+  ...callstackConfigReact,
+  {
+    files: [
+      'eslint.config.js',
+      'example/**/*.{js,ts,tsx}',
+      'example/*.{js,ts,tsx}',
```

Comment:
Is this some glob incompatibility of eslint? Shouldn't line 11 already handle that case?

Reply:
you are right, will update it